docs/runbook/heroku.md

```diff
@@ -0,0 +1,31 @@
+# Deploying to heroku
+
+The app has been designed to easily deploy to heroku.
+
+## Pipelines
+
+We recommend [creating a pipeline](https://devcenter.heroku.com/articles/pipelines) with a staging and production app.
+
+Each app requires the [heroku/node.js buildpack](https://devcenter.heroku.com/articles/buildpacks) and uses the following addons: [Heroku Postgres](https://elements.heroku.com/addons/heroku-postgresql), [Heroku Scheduler](https://elements.heroku.com/addons/scheduler).
```

Comment:
Might be worth highlighting that they should probably use a paid for dyno, and they'll need to add a postgress add on